LicenseOwners/Pages/Index.cshtml

```diff
@@ -33,11 +35,18 @@
                             <td>@businessLicenseOwner.BusinessActivity</td>
                             <td>@businessLicenseOwner.OwnerFirstName</td>
                             <td>@businessLicenseOwner.OwnerLastName</td>
+                             @if (businessLicenseOwner.OwnerFirstName != null)
+                                     {
                             <td>@businessLicenseOwner.OwnerTitle</td>
-                            
-                        </tr>
+                                     }
+                             else
+                                    {
+                            <td>  </td>
```

Comment:
Ahhhhh... this doesn't match, as there are three TDs above, and only one here.  To fix, add colspan="3" attribute.  But... do you even need to show anything if the if test fails?  Why not just skip the row an dprint nothing?  

I also don't see an open <tr> tag, but that may just be the truncated view.